tests/plotting/test_plotting.py

```diff
@@ -4548,6 +4549,159 @@ def _create_object(_direction=None):
     plot.show()
 
 
+@pytest.fixture
+def _allow_empty_mesh():
+    # setup
+    flag = pv.global_theme.allow_empty_mesh
+    pv.global_theme.allow_empty_mesh = True
+    yield
+    # teardown
+    pv.global_theme.allow_empty_mesh = flag
+
+
+@pytest.fixture
+def _show_edges():
+    # setup
+    flag = pv.global_theme.show_edges
+    pv.global_theme.show_edges = True
+    yield
+    # teardown
+    pv.global_theme.show_edges = flag
+
+
+@pytest.mark.usefixtures('_allow_empty_mesh', '_show_edges')
+@pytest.mark.parametrize(
+    ('select_inputs', 'select_outputs'),
+    [(None, None), (None, 2), (2, 2)],
+    ids=['in_None-out_None', 'in_None-out_2', 'in_2-out_2'],
+)
+@pytest.mark.needs_vtk_version(9, 3, 0)
+def test_contour_labels_boundary_style(
```

Comment:
This test is made possible by this PR (it uses `smoothing=False`), but is otherwise not really testing the smoothing parameter. This test provides additional coverage for boundary style.

We could split this as a separate PR if desired. There are other few misc test coverage changes from #6732 that could probably be lumped in as well.

Reply:
I'm fine as is, but if you want to do it, I'm happy to review new PR.

Reply:
Ok let's keep it as-is